src/common-principles.md

```diff
@@ -504,16 +504,20 @@ TSV files MUST be in UTF-8 encoding.
 
 Example:
 
-```Text
-onset   duration    response_time   trial_type        trial_extra
-200     20.0        15.8            word              中国人
-240     5.0         17.34e-1        visual            n/a
+```tsv {linenums="1"}
+onset	duration	response_time	trial_type	trial_extra
+200	20.0	15.8	word	中国人
+240	5.0	17.34e-1	visual	n/a
 ```
 
 !!! warning "Attention"
 
     The TSV examples in this document (like the one above this note) are occasionally
-    formatted using space characters instead of tabs to improve human readability.
+    formatted with the addition of the row indexes
+    as first column.
+    Those indexes are presented for visual reference and
+    are not part of the tabular data file's content.
+
     Directly copying and then pasting these examples from the specification
     for use in new BIDS datasets can lead to errors and is discouraged.
```

Comment:
Sorry, I meant that in the `tsv` blocks, copy-pasting is now fine. We can remove the discouragement.

Also changing indexes to indices.

```suggestion
    formatted with the addition of the row indices as first column.
    Those indices are presented for visual reference and
    are not part of the tabular data file's content.
```